README.md

```diff
@@ -2,6 +2,27 @@
 
 Preview components in isolation with documentation, annotations and breakpoint controls.
 
+## Branding
+
+You can change the demo-viewer to look a little more like your own company brand.
+
+On the initial config for the demo-viewer you can pass a path to your logo.
+
+	window.demoViewer({
+        modules: [
+        	{ logo: '/path/to/logo' },
```

Comment:
makes more sense to me if logo is on the same level as modules (not a module itself).